Src/DependencyCollector/Shared.Tests/HttpDependenciesParsingTelemetryInitializerTest.cs

```diff
@@ -262,5 +262,58 @@ private void HttpDependenciesParsingTelemetryInitializerConvertsQueues(string op
             Assert.AreEqual(parsedUrl.Host, d.Target, operation);
             Assert.AreEqual(accountName + "/" + queueName, d.Name, operation);
         }
+
+        [TestMethod]
+        public void HttpDependenciesParsingTelemetryInitializerConvertsServices()
+        {
+            var testCases = new List<string[]>()
+            {
+                ////
+                //// copied from https://msdn.microsoft.com/en-us/library/azure/dd179423.aspx 10/19/2016
+                ////
+
+                new string[4] { "PUT", "https://microsoft.com/test.asmx", RemoteDependencyConstants.WebService, "/test.asmx" },
+                new string[4] { "GET", "https://microsoft.com/test.svc", RemoteDependencyConstants.WcfService, "/test.svc" },
+                new string[4] { "POST", "https://microsoft.com/test.asmx/myargument", RemoteDependencyConstants.WebService, "/test.asmx" },
+                new string[4] { "HEAD", "https://microsoft.com/test.svc/myarguments", RemoteDependencyConstants.WcfService, "/test.svc" },
+            };
```

Comment:
Is there a way we can detect Web API services that don't have classic extensions in their URL's?